crates/viewer/re_ui/src/notifications.rs

```diff
@@ -0,0 +1,393 @@
+use std::time::Duration;
+
+use egui::hex_color;
+use jiff::{Unit, Zoned};
+pub use re_log::Level;
+
+use crate::icons;
+use crate::UiExt;
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+enum NotificationLevel {
+    Info,
+    Warning,
+    Error,
+    Success,
```

Comment:
Please document the difference between `Info` and `Success`

Reply:
I don't know what the difference is, it came from `ToastKind` which was also undocumented. I guess someone wanted to use a different color?